src/script/api/Doxyfile_AI.in

```diff
@@ -91,7 +90,7 @@ WARN_LOGFILE           =
 #---------------------------------------------------------------------------
 # configuration options related to the input files
 #---------------------------------------------------------------------------
-INPUT                  = . $(GENERATED_API_DIR)
+INPUT                  = . ${FIND_VERSION_BINARY_DIR}/script/api
```

Comment:
`FIND_VERSION`? Shouldn't this be `CMAKE_SOURCE_DIR`?

Reply:
`.` is actually `CMAKE_CURRENT_SOURCE_DIR` (via `WORKING_DIRECTORY` in the target). And we also need generated files (`script_window.hpp` mainly) and `FindVersion.cmake` gets it via target command